rust/cubesql/src/compile/mod.rs

```diff
@@ -1545,9 +1545,15 @@ impl QueryPlanner {
 
             Ok(QueryPlan::CubeSelect(StatusFlags::empty(), builder.build()))
         } else {
+            // @TODO Can we do a fuzzy search on the cubes we have and suggest possible names?
```

Comment:
@ovr WDYT?

Reply:
It's a great idea, let's create a task for it 👍